cmd/updater/version_test.go

```diff
@@ -29,10 +29,10 @@ func TestGetVersion(t *testing.T) {
 	partitiontest.PartitionTest(t)
 	t.Parallel()
 
-	testValidVersion(t, "algonode_update_0.1.0.log", uint64(0x00010000))
-	testValidVersion(t, "algo_update_0.1.0", uint64(0x00010000))
-	testValidVersion(t, "algo_update_65535.1.0", uint64(0xFFFF00010000))
-	testValidVersion(t, "algo_update_65535.65535.65535", uint64(0xFFFFFFFFFFFF))
+	testValidVersion(t, "algonode_update_0.1.0.log", uint64(0x01000000))
+	testValidVersion(t, "algo_update_0.1.0", uint64(0x01000000))
+	testValidVersion(t, "algo_update_65535.1.0", uint64(0x00FFFF000001000000))
+	testValidVersion(t, "algo_update_65535.65535.65535", uint64(0x00FFFF00FFFF00FFFF))
```

Comment:
May as well add the test
```suggestion
	testValidVersion(t, "algo_update_65535.65535.65535", uint64(0x00FFFF00FFFF00FFFF))
	testValidVersion(t, "algo_update_16777216.16777216.16777216", uint64(0xFFFFFFFFFFFFFFFFFF))
```

Reply:
this number exceeds max uint64
maybe allocate 24bits for the patch and 16+16 for major/minor?

Reply:
16/16/24 seems like a good option.

Reply:
added